homeassistant/components/ebusd/sensor.py

```diff
@@ -0,0 +1,107 @@
+"""
+Support for Ebusd daemon for communication with eBUS heating systems.
+
+For more details about ebusd deamon, please refer to the documentation at
+https://github.com/john30/ebusd
+"""
+
+import logging
+import datetime
+
+from homeassistant.helpers.entity import Entity
+
+DEPENDENCIES = ['ebusd']
+
+DATA_EBUSD = 'EBUSD'
```

Comment:
Don't redefine this, instead do something like: `from homeassistant.components.ebsud.const import DOMAIN as EBUSD_DOMAIN` and use that instead.